parser-native/src/main/java/ru/urururu/sanity/cpp/NativeSourceRangeFactory.java

```diff
@@ -14,8 +15,8 @@
  * @author <a href="mailto:dmitriy.g.matveev@gmail.com">Dmitry Matveev</a>
  */
 @Component
-public class SourceRangeFactory implements ParserListener {
-    private static final Logger LOGGER = LoggerFactory.getLogger(SourceRangeFactory.class);
+public class NativeSourceRangeFactory extends SourceRangeFactory<SWIGTYPE_p_LLVMOpaqueValue> implements ParserListener {
+    private static final Logger LOGGER = LoggerFactory.getLogger(NativeSourceRangeFactory.class);
 
     private static final int DW_TAG_file_type = 786473;
     private static final int DW_TAG_lexical_block = 786443;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00115)